Resolve wire issues and remove difficult colors

neersighted opened the pull request:
:us: Remove British colors :us:

neersighted committed:
Remove dark/british colors from wires

Fixes #14901

Iamgoofball commented:
:us:

neersighted commented:
Ah Travis, we hardly knew 'ye.

GunHog commented:
![american-eagle-flag-wallpaper-hd](https://cloud.githubusercontent.com/assets/4955510/12568367/f96368a0-c38a-11e5-9283-3dbea791c07d.jpg)

PKPenguin321 commented:
:us: :gun: :us:
:+1: for freedom colors instead of degenerate colours

ChangelingRain commented:
:uk:

Anonmare commented:
:uk:
Down with the colonial scum tbh
:uk:

neersighted committed:
Fix a number of wire issues.

Fix wires being shuffled twice (they should only be randomized once). We
now shuffle and cache instead of shuffling the cache.

Fix syndicate bomb messages (bomb, not loc should show messages).

Explicitly copy lists when we create wires as to avoid any chance of
modifying the global color list. No bugs here but there is a posibility
if admins varedit wires.

Fix training bombs generating duplicate wires by adding a real
shuffle proc that clears the wire list first (Fixes #14915).

optimumtact commented:
:gb: FOR QUEEN AND COUNTRY DENY THIS PR

KazeEspada commented:
@optimumtact dont you luve in a former colony?

neersighted committed:
Remove old define

RemieRichards commented on code/modules/admin/verbs/mapping.dm:
```diff
@@ -174,7 +174,7 @@ var/intercom_range_display_status = 0
 			continue
 		count++;
-	usr << "There are [count]/[MAX_FLAG] instances of non-turfs in the world."
+	usr << "There are [count]/65535 instances of non-turfs in the world."
```
this is incorrect anyways.

The following are limited to 16,777,216 instances at runtime:

/obj
/mob
/area
/image
Appearances (this is something internal with BYOND)
Unique overlay and underlay lists [This is an internal type called an ID array.]
User defined datums
[Consider turfs unofficially on this list.]

neersighted committed:
Remove unused mapping verb

RemieRichards commented:
See, @neersighted I didn't want you to just swap the numbers, because that's incorrect, you can have 16 mil turfs, and 16 mil PER OTHER TYPES, that's like, 7*16Mil or 112Mil non-turf instances.

ike709 commented:
![image](https://cloud.githubusercontent.com/assets/5714543/12596168/4bc20322-c444-11e5-86b7-5b300a00a2e4.png)
